proxies/live/apiproxy/targets/ers-target.xml

```diff
@@ -9,17 +9,26 @@
       </Step>
       <Condition>oauthV2.OauthV2.VerifyAccessToken.failed</Condition>
     </FaultRule>
+    <FaultRule name="spike_arrest_fault">
+       <Step>
+            <Name>AssignMessage.SpikeArrestErrorResponse</Name>
+       </Step>
+       <Condition>(fault.name Matches "SpikeArrestViolation")</Condition>
```

Comment:
Should this condition be using a flow variable as used above? Looks like a `failed` variable is set should a SpikeArrest policy fail (https://docs.apigee.com/api-platform/reference/policies/spike-arrest-policy#variables) so perhaps this condition should be something like
`ratelimit.SpikeArrest.failed or ratelimit.SpikeArrest.Binary.failed`

Reply:
I thought of using the ratelimit.SpikeArrest variables. But used fault name check instead. Because if we add more spike address policies for various endpoints in future, we have keep checking this variables and modify the fault rule. or someone adds SpikeArrest Policy and forgot to add variable in the fault rule and may expose the Apigee errors. So I thought it would be better use the fault name check. Hope this is ok for you?